doc/README.md

```diff
@@ -0,0 +1,83 @@
+# Network Distributed Architecture
+
+## Websockets - Default Response for New Connection
+
+```mermaid
+sequenceDiagram
+    autonumber
+    box Process Id p1
+    participant runtime@p1   
+    end
+
+    box Process Id p2
+    participant webgui@p2
+    participant runtime@p2
+    end
+
+    Note right of runtime@p1: Client runtime@p1 opens a <br/>websocket to remote webgui
+
+    runtime@p1->>webgui@p2: Connect<br/>ws://localhost:8888/api/messages?user=root&pwd=pwd&session_id=2309adf3dlkdk&id=p1
+
+    Note left of webgui@p2: Remote webgui@p2 and runtime@p2 attempt to subscribe <br/> to the describe method of the runtime@p1
+    webgui@p2->>runtime@p1: addListener describe
+    Note left of webgui@p2: runtime@p2 sends a describe request to runtime@p1
+    webgui@p2->>runtime@p1: describe 
+    Note right of runtime@p1: runtime@p1 responds with a describe response
+    runtime@p1->>webgui@p2: onDescribe 
+    Note left of webgui@p2: Based on the results of the describe,<br/> more querying and subscriptions can be processed
+
+
+    %% opt Server Add Listener, Describe and Reserve
+    %% end
+
+
+```
+### 1 Connection
```

Comment:
Should explain what the query string does, I thought the `id` parameter was important for connection but a quick look through `WebGui.handle()` doesn't show any usage of it

Reply:
Valid point, initially I thought its how one process could identify itself, but registration does that anyway.  Additionally, I haven't worked out the details between where authentication and authorization should be.  Currently, its part of the Security service, but not really utilized yet.